pilot/lib/pilot/build_manager.rb

```diff
@@ -81,7 +79,12 @@ def list(options)
         config[:app_identifier] = UI.input("App Identifier: ")
       end
 
-      builds = app.all_processing_builds + app.builds
+      if config[:app_platform].to_s.length == 0
+        config[:app_platform] = ask("App Platform (ios, appletvos, osx): ")
+      end
```

Comment:
In fact, can't we just call `fetch_app_platform` here as well, since that already has the `ask` invocation in it.